.azure-pipelines/client.test.live.yml

```diff
@@ -49,7 +51,7 @@ jobs:
           BLOB_STORAGE_ACCOUNT_NAME: $(java-storage-test-blob-account-name)
           BLOB_STORAGE_ACCOUNT_KEY: $(java-storage-test-blob-account-key)
           AZCONFIG_CONNECTION_STRING: $(java-azconfig-test-connection-string)
-
+          AZURE_SERVICEBUS_JAVA_CLIENT_TEST_CONNECTION_STRING: ${java-servicebus-test-connection-string}
```

Comment:
This should not be curly braces - the rest of the variables are not like this.

Reply:
nit: I don't think JAVA_CLIENT_TEST is necessary. 🤔 But I guess you'd also have to update the calling code with the new name.

Reply:
updated